### PR TITLE
[show-tech] installing ansible community and updating README.md

### PR DESCRIPTION
**Pull request by AyliD**
Signed-off-by: Ayelet Regev Dabah <ayelet.regev@gmail.com>

## Summary

Resolving chicken and egg issue identified on vagrant that playbook that installs this community cannot parse the playbook till community is installed.
Adding installation to physical installations script and updating README.md for vagrant.

## Test Plan

```
$ bash lte/gateway/deploy/agw_install.sh
```

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by karthiksubraveti — APPROVED** [review]

File: lte/gateway/deploy/agw_install.sh [hunk 1]
```diff
@@ -106,6 +106,7 @@ if [ "$MAGMA_INSTALLED" != "$SUCCESS_MESSAGE" ]; then
   apt-get -y install curl make virtualenv zip rsync git software-properties-common python3-pip python-dev
   alias python=python3
   pip3 install ansible
+  ansible-galaxy collection install community.general
```
Comment (tmdzk) [inline lte/gateway/deploy/agw_install.sh]: Let's wrap ansivle-galaxy collection install in the base role of agw installation.

File: show-tech/README.md [hunk 2]
```diff
@@ -5,6 +5,11 @@ On your GW host make sure you have the following packages:
 $ apt-get install git ansible -y
 ```
 
+On vagrant you will also need to install community collection
+```
+$ sudo ansible-galaxy collection install community.general
```
Comment (tmdzk) [inline show-tech/README.md]: There is an ansible role that provision the Vagrant box, let's add it there.

**Review by tmdzk — CHANGES_REQUESTED** [review]

**Review by karthiksubraveti — COMMENTED** [review]
We should also make changes in show-tech playbooks to remove the installation of ansible community right?

**Review by AyliD — COMMENTED** [review]
Moved instructions to playbooks @tmdzk

**Review by tmdzk — APPROVED** [review]

**Review by pshelar — APPROVED** [review]
LGTM

**Pull request merged**